strapi/strapi
# Api token get content api routes

### What does it do?

Adds an admin route to get content api routes

### How to test it?

- Run a Strapi app and try to get the routes using `admin/content-api/routes`
it should return a structure similar to this 
```js
{
data:{
    'api::address': [
      {
        method: 'GET',
        path: '/api/addresses',
        handler: 'api::address.address.find',
        config: {
          auth: false,
        },
        info: {
          apiName: 'address',
          type: 'content-api',
        },
      },
      {
        method: 'GET',
        path: '/api/addresses/:id',
        handler: 'api::address.address.findOne',
        config: {
          auth: {
            scope: ['api::address.address.findOne'],
          },
        },
        info: {
          apiName: 'address',
          type: 'content-api',
        },
      },
    ],
   }
 }
  ```
